Content update

emileighoutlaw opened the pull request:
Updates privacy policy language, per our re-writes and approval from Interior's Departmental Privacy Officer.

emileighoutlaw committed:
Content update

Updates privacy policy language, per our re-writes and approval from Interior's Departmental Privacy Officer.

OpenGlobe commented:
I think we can also remove (or replace) this link:

http://www.ftc.gov/bcp/conline/edcams/kidzprivacy/index.html

Appears to be broken.

khandelwal commented on ekip/everykid/templates/legal/privacy.html:
```diff
@@ -6,58 +6,29 @@
+ 
+<p>We collect general information about our visitors so we can improve our website. 
+If you want more information, see the <a href="http://www.doi.gov/privacy.cfm">Department of the Interior Privacy Policy</a>. 
+Also see the <a href="http://www.ftc.gov/bcp/conline/edcams/kidzprivacy/index.html">Government Children's Privacy Policy </a>.</p>
```
This link doesn't work. So we'll have to find a new version or remove it.

OpenGlobe commented:
I think I have an updated link - just a sec

OpenGlobe committed:
Removed bad link

Removed the bad link.  This wasn't provided but was in an earlier draft, so I'm taking it out.

emileighoutlaw commented:
woot!

On Fri, Aug 28, 2015 at 2:31 PM, Shashank Khandelwal <
notifications@github.com> wrote:

> Merged #108 https://github.com/18F/ekip-api/pull/108.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/18F/ekip-api/pull/108#event-395352014.
